pkg/question/question.go

```diff
@@ -719,12 +769,17 @@ func AskSubnetPlaceholder(h *ec2helper.EC2Helper) (*string, error) {
 	indexedOptions := []string{}
 
 	// Add availability zones to the data for table
+	var defaultOptionRepr *string
+	var defaultOptionValue *string
```

Comment:
Since both of these variables are always assigned the same value, can one be removed?